src/pages/CropDetailsPage.js

```diff
@@ -74,26 +73,30 @@ export default function CropDetailsPage({
 }
 const BackButton = styled(Button)`
   position: fixed;
-  padding: 15px 25px;
+  padding: 8px 12px;
+  left: 10px;
+  top: 10px;
   z-index: 1;
 `
 const DetailedView = styled.section`
   img {
     width: 100%;
     height: auto;
-    position: sticky;
+    position: fixed;
     top: 0;
     z-index: -1;
   }
 `
 
 const Information = styled.div`
   background: white;
-  padding: 10px;
+  padding: 10px 20px;
   border-top-left-radius: 10px;
   border-top-right-radius: 10px;
```

Comment:
Maybe it´s easier to use the following? Should be the same result. 
`border-radius: 10px 10px 0px 0px;`